README.md

```diff
@@ -185,9 +185,9 @@ rails generate mountain_view:extra_pages
 A style guide will be automatically generated. This style guide never falls behind and it reflects your components in their latest version.
 
 #### Setting up the style guide
-1) Add the following line to your `routes.rb` file.
+1) Add the following line to your `routes.rb` file. `/mountain_view` can be any name you want.  `mv_style_guide` is expected in order to find the engine routes.  
```

Comment:
I'm not happy with this wording, but I wanted to get something in as a starting point.

Reply:
It's definitively a good starting point. Few suggestions:

- "`/mountain_view` can be any name you want." => "`/mountain_view` can be any path you want."
- "`mv_style_guide` is expected in order to find the engine routes. " => "Adding `as: :mv_style_guide` is needed for the style guide to provide a link to the main page." (Not very happy with this one either, but trying to highlight the benefir rather than the technical details)
- And maybe we can add this as sub-points after the code example so it's more clear what we are referring to.